### ISSUE TITLE
Change Password Reset Email Address

### ISSUE THREAD

**Issue by katyedwards** [healthlocker/healthlocker#807]
+ [x] change to healthlocker@slam.nhs.uk

Currently the reset password email comes from: healthlocker.test@gmail.com

Should this be updated to a SLaM email, or the healthlocker@slam address?

**Comment by MollieCourtenay** [comment]
@iteles can this email be updated?

**Comment by iteles** [comment]
@MollieCourtenay We can get this done, I don't think we should go live with this one. 

Which email account would you like it to come from?

**Comment by MollieCourtenay** [comment]
healthlocker@slam.nhs.uk

**Comment by nelsonic** [comment]
in order to update this we will need someone who has _access_ to this email/inbox to forward a verification code that will be sent to the email. do you know who "owns" it?

**Comment by MollieCourtenay** [comment]
Adele does @nelsonic

**Comment by RobStallion** [comment]
We have triggered the email with the verification code. Adele just needs to verify this. Will message her on slack.

With regards to the staging server, it's a fairly quick fix but we just need confirmation that we can access the staging server to change the variable. @nelsonic, would that be ok to do today?

**Comment by katyedwards** [comment]
@RobStallion have emailed Adele too - not sure if she is looking at slack much.  Thanks!

**Comment by RobStallion** [comment]
@katyedwards The email has still not been verified. Can you check in with Adele and see if she has received the email yet and ask her to click the link if she has? Also just ask her to check and make sure that it hasn't gone into her junk/spam mail. Thanks

**Comment by katyedwards** [comment]
@RobStallion yes will ask her.  Do you know what address it should have come from in case it has been blocked by slam it before getting to her?  

Will be quicker if I can ask her to check both at the same time!

**Comment by RobStallion** [comment]
@katyedwards it should come from _no-reply-aws@amazon.com_.

Just tested it with one of the office emails and we received it in a few seconds so seems to be working ok.

**Comment by katyedwards** [comment]
Hi @RobStallion sorry - can you try to resend the email to the healthlocker email address again.

Thanks!

**Comment by RobStallion** [comment]
@katyedwards. We have sent it again. I also sent an email from my dwyl account just to test that we have the correct email address. Can you ask Adele to take a look at it when she has a moment?

**Comment by katyedwards** [comment]
@RobStallion yep will do!

**Comment by markwilliamfirth** [comment]
Email has been verified. This issue is now linked to #894

**Comment by katyedwards** [comment]
Just to clarify, as far as you are aware is this issue done and sorted, just pending the fixing of #894 so we can test it?

**Comment by markwilliamfirth** [comment]
@katyedwards yes exactly. This issue can now be tested and closed. It links to the new issue mentioned in your post here https://github.com/healthlocker/healthlocker/issues/894#issuecomment-313350943

As #894 is now solved (module is now available) this has now been closed in favour #905 (which is linked)

**Comment by markwilliamfirth** [comment]
Updating this issue as this is currently being blocked by #905

**Comment by katyedwards** [comment]
@RobStallion Stuart asked me to test and the forgot password link worked and I received an email with the reset link.

So I think you can work on this issue again.

FYI - the reset link directed me to Healthlocker.uk not staging.healthlocker.uk so I got an HL error and was not able to reset my password.

![screenshot 2017-07-13 11 28 57](https://user-images.githubusercontent.com/24604903/28162308-799924e6-67be-11e7-9287-ee7c587da1f9.png)

**Comment by RobStallion** [comment]
Copy of email that is sent for password reset. 

![image](https://user-images.githubusercontent.com/15571853/28163029-4c43162a-67c1-11e7-88a2-29228ec3a011.png)

**Comment by katyedwards** [comment]
Yes, all working.
